presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -82,6 +86,12 @@ public String getTableName()
         return tableName;
     }
 
+    @JsonProperty
+    public OptionalLong getLimit()
+    {
+        return limit;
+    }
```

Comment:
consider adding to toString. This should let EXPLAIN show when limit was pushed down to the connector level

Reply:
Good idea. Updated

Reply:
I also fixed column handle rendering. We now get a plan that looks like this:
```
Fragment 1 [SOURCE]
    Output layout: [orderkey]
    Output partitioning: SINGLE []
    Grouped Execution: false
    - TableScan[postgresql:tpch.orders tpch.orders limit=10, grouped = false] => [orderkey:bigint]
            Estimates: {rows: ? (?), cpu: ?, memory: 0.00, network: 0.00}
            orderkey := orderkey:bigint:int8
```